sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -839,6 +856,32 @@ export class ServiceBusAtomManagementClient extends ServiceClient {
     if (isUpdate) {
       webResource.headers.set("If-Match", "*");
     }
+
+    const queueOrSubscriptionFields = entityFields as
+      | InternalQueueOptions
+      | InternalSubscriptionOptions;
+    if (
+      queueOrSubscriptionFields.ForwardTo ||
+      queueOrSubscriptionFields.ForwardDeadLetteredMessagesTo
+    ) {
+      const token = (await this.sasTokenProvider.getToken(this.endpoint)).token;
+      const urlPrefix = `sb://${this.endpoint}/`;
+      if (queueOrSubscriptionFields.ForwardTo) {
+        webResource.headers.set("ServiceBusSupplementaryAuthorization", token);
+        queueOrSubscriptionFields.ForwardTo = isAbsoluteUri(queueOrSubscriptionFields.ForwardTo)
+          ? queueOrSubscriptionFields.ForwardTo
+          : urlPrefix.concat(queueOrSubscriptionFields.ForwardTo);
```

Comment:
We can simplify by skipping assignment if absolute url is given
```javascript
if (!isAbsoluteUri(...)) {
     queueOrSubscriptionFields.ForwardTo = // concat here
}
```

Reply:
Updated ternary check -> if check